rabbit-escape-ui-android/app/src/main/java/rabbitescape/ui/android/AndroidCanvas.java

```diff
@@ -40,6 +43,24 @@ public void drawLine(
         float startX, float startY, float stopX, float stopY, AndroidPaint paint
     )
     {
-        this.canvas.drawLine( startX, startY, stopX, stopY, paint.paint );
+        this.canvas.drawLine(startX, startY, stopX, stopY, paint.paint);
+    }
+
+    @Override
+    public void drawFilledPoly( int[] xs, int[] ys, AndroidPaint paint )
```

Comment:
I would still prefer to implement a small part of the Android interface than make our own 3rd interface that is unlike either the swing or Android one. It makes it easier to understand where the logic is, and hopefully makes it easier to share code between the 2 implementations.